r/redisearch/build_info.json

```diff
@@ -2,15 +2,21 @@
   "maintainer": "Vinod.K1@ibm.com",
   "package_name": "redisearch",
   "github_url": "https://github.com/RediSearch/RediSearch",
-  "version": "v2.4.0",
+  "version": "v2.8.13",
   "default_branch": "master",
   "package_dir": "r/redisearch",
   "docker_cmd": "docker build -t ${package_name}:$PACKAGE_VERSION ${dir}",
-  "build_script": "redisearch_ubi8.sh",
+  "build_script": "redisearch_ubi_9.3.sh",
+  "validate_build_script": "true",
+  "use_non_root_user": "false",
+  "docker_build": "true",
   "v2.0.*": {
     "dir": "2.0.0-ubi8"
   },
   "v2.4.*": {
     "dir": "2.4.16-ubi8"
```

Comment:
also here do we need to mention earlier build script or new script works for earlier version as well